news/6384.build.rst

```diff
@@ -0,0 +1,2 @@
+The bootloader can be force compiled during pip install by setting the
+environment variable `PYINSTALLER_COMPILE_BOOTLOADER`.
```

Comment:
This one will require double-backtick quotes.

Reply:
Ok, but not the `:envvar:` like in the main documentation?

Reply:
No, `:envvar:`, `:func:`, `:file:`, etc. require a single backtick. But if you don't use any of those, a single backtick will produce italicized text, whereas double backticks will produce the bold-darkblue text that we usually use for the emphasis.

Although now that you mention it, the reference to environment variable in the news fragment could also be changed to 
```
:envar:`PYINSTALLER_COMPILE_BOOTLOADER`
```
instead of having it use double-backtick quotes as I originally suggested.

Reply:
Done ✔️

Reply:
Unfortunately, for the fancy cross reference markup to do anything more than issue an unfriendly warning, it needs some API reference style definition to cross reference to. As you can see [here in readthedoc's PR preview](https://pyinstaller--6384.org.readthedocs.build/en/6384/CHANGES.html#bootloader-build), it has just defaulted to regular literal markup i.e. equivalent to quoting with double backticks. We *could* give it what it wants and wrap your little new paragraph in the docs like so:
```diff
diff --git i/doc/bootloader-building.rst w/doc/bootloader-building.rst
index 5f9e55a8..e021d978 100644
--- i/doc/bootloader-building.rst
+++ w/doc/bootloader-building.rst
@@ -43,9 +43,11 @@ machine, run::
 If this reports an error, read the detailed notes that follow,
 then ask for technical help.
 
-By setting the environment variable :envvar:`PYINSTALLER_COMPILE_BOOTLOADER`
-the pip_ setup will attempt to build the bootloader for your platform, even
-if it is already present.
+.. envvar:: PYINSTALLER_COMPILE_BOOTLOADER
+
+    By setting the environment variable :envvar:`PYINSTALLER_COMPILE_BOOTLOADER`
+    the pip_ setup will attempt to build the bootloader for your platform, even
+    if it is already present.
 
 Supported platforms are
```
But it looks completely out of place having a one entry API reference style in the middle of that page so I'd suggest we just stick to using double backticks.

Reply:
Done.